eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -93,9 +93,8 @@
   <suppress checks="IllegalImport" files="com.azure.messaging.servicebus.IntegrationTestBase.java"/>
   <suppress checks="IllegalImport" files="com.azure.messaging.servicebus.TracingIntegrationTests.java"/>
   <suppress checks="IllegalImport" files="com.azure.messaging.servicebus.LoggingSpanProcessor.java"/>
-  <suppress checks="IllegalImport" files="com.azure.messaging.eventhubs.TracingIntegrationTests.java"/>
-  <suppress checks="IllegalImport" files="com.azure.messaging.eventhubs.PublishEventsTracingWithCustomContextSample.java"/>
-  <suppress checks="IllegalImport" files="com.azure.messaging.eventhubs.EventProcessorWithTracingClientSample.java"/>
+  <suppress checks="IllegalImport" files="com.azure.messaging.eventhubs..*Sample.*.java"/>
```

Comment:
Since it's using regex, the `.` is "match anything" rather than the period we want to match against. I mean, it doesn't change anything wrt to being able to match it. :) But I thought it was funny.